### PR TITLE
Created new branch for new work on add four cards.

### PR DESCRIPTION
**Pull request by IStallcup**
Added more stuff to facilitate adding four cards.

### CODE REVIEW

**Comment by kannabanana** [comment]
You should work on your commenting and remember to move the last function before you call it - good code practice!

File: Assignment1/src/test/java/models/cardModelsTest.java [hunk 1]
```diff
@@ -31,6 +31,24 @@ public void compareCards()
...
         card mini_card = new card(9,"heart");
         assertEquals(1, mini_card.compare(num,suit));
+        }
+
+    @Test
+    public void compareCardsFail()
+    {
+        int num = 2;
+        String suit = "heart";
+        card compareCard = new card(2, "spades");
```
Comment (athai) [inline Assignment1/src/test/java/models/cardModelsTest.java]: Change spades -> spade

The deck class initializes the suit as spade.

**Comment by athai** [comment]
lgtm. +1

**Comment by leel8on** [comment]
:+1:  lgtm

**Pull request merged**